superset/views.py

```diff
@@ -1253,6 +1254,23 @@ def slice(self, slice_id):
         viz_obj = self.get_viz(slice_id)
         return redirect(viz_obj.get_url(**request.args))
 
+    @log_this
+    @has_access_api
+    @expose(
+        "/update_explore/<datasource_type>/<datasource_id>/", methods=['POST'])
+    def update_explore(self, datasource_type, datasource_id):
+        form_data = json.loads(request.form.get('data'))
+        error_redirect = '/slicemodelview/list/'
+        try:
```

Comment:
ahhh, i've been equating querying with saving as well, my bad. sorry for the confusion. will revise my comments.